dockerfiles/idp_prod.Dockerfile

```diff
@@ -148,6 +213,7 @@ COPY certs.example $RAILS_ROOT/certs
 COPY config/service_providers.localdev.yml $RAILS_ROOT/config/service_providers.yml
 
 # Precompile assets
+RUN apt-get install -y make
```

Comment:
can we move this up into the big install block?

We could also move asset compilation into the build stage as well and not install any yarn/javascript I think.

Reply:
We can, though it won't make any significant difference.  

I didn't move the asset compilation up, because I don't know where all the assets get put.  It looked like there were a lot of assets directories in there.  Is there a single one that I can copy back over?

Reply:
I think it should just be the `public/` folder